utils/motorcontroller/defaultconfig/NeoDefaults.java

```diff
@@ -0,0 +1,27 @@
+package frc.team670.mustanglib.utils.motorcontroller.defaultconfig;
+
+import frc.team670.mustanglib.utils.motorcontroller.MotorConfig.SparkMAXFactory.Config;
+
+public class NeoDefaults {
+    // for velocity control, all we need is frame 1, which has the velocity value
+
```

Comment:
I think we probably want a comment in this file that explains the data that each status frame carries. That'll make it easier to understand why we're turning some up or down